Changelog.md

```diff
@@ -9,8 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Added an end-to-end test for validity of generated `all.css` file.
-  #199 by @MaxDesiatov.
+- Added end-to-end tests for generate command-line interface.
```

Comment:
```suggestion
- Added end-to-end tests for `generate` command-line interface.
```

Reply:
On second thought, better to remove mention of `generate` entirely, since we're now testing all subcommands.